dataaccess.js

```diff
@@ -22,10 +22,9 @@ var rankingsDisabled = false;
 
 var connection = new cradle.Connection();
```

Comment:
After our discussion about caching everything, we might want to limit the cache size. You pass a maximum number of objects to cache here:

    new cradle.Connection({cacheSize: 10000})